docs/contribute/core.qmd

```diff
@@ -91,31 +87,12 @@ pkg> add Revise TestEnv
 Revise.jl is a library that allows you to modify code and use the changes without restarting Julia.
 You can let it start automatically by following these [instructions](https://timholy.github.io/Revise.jl/stable/config/#Using-Revise-by-default-1).
 
-## Clone Ribasim
-
-In order to have the Ribasim repository locally available, you can clone it with Git.
-Git can be installed from [git-scm.com](https://git-scm.com/downloads).
-Once installed, run the following command at a directory of your choice:
-
-In order to have the Ribasim repository locally available, run the following command at a directory of your choice:
-
-```bash
-git clone https://github.com/Deltares/Ribasim.git
-```
-
-To continue with the following steps, make the root of the repository your working directory by running
-
-```bash
-cd Ribasim
-```
```

Comment:
Is there a reason why you removed this?
Many people are familiar with git nowadays, but I don't think it's general knowledge just yet.

Reply:
Then I will put it in the index.qmd instead.